pkg/vault/contracts/BatchRouterHooks.sol

```diff
@@ -550,8 +588,8 @@ contract BatchRouterHooks is BatchRouterCommon {
 
         (uint256[] memory exactAmountsOut, ) = _getSingleInputArrayAndTokenIndex(pool, tokenOut, stepExactAmountOut);
 
-        // The Router is always an intermediary in this case. The Vault will burn tokens from the Router, so
-        // the Router is both owner and spender, which doesn't require approval.
+        // Router is always an intermediary in this case. The Vault will burn tokens from the Router, so
+        // Router is both owner and spender (which doesn't need approval).
```

Comment:
I never know when we use the article and when we don't, but I had the impression that "The Router" here was correct (especially looking at other comments that always start with "The user, The last step, The buffer", etc.)